Automattic/woocommerce-payments
# Create WooPay user from trial subscription

#### Changes proposed in this Pull Request









* Allow to create a WooPay user when buying a trial subscription.

This PR needs to be tested alongside 2350-gh-Automattic/woocommerce-payments-server and it's blocked by #4500.

#### Testing instructions





* Buy a trial subscription on the merchant store.
* The "Save my information for faster checkouts" field show be shown, click on it to create a new WooPay user.
* After the success page, buy any new product and type the last e-mail address on checkout page, the WooPay OTP modal should be shown.

-------------------


**Post merge**
